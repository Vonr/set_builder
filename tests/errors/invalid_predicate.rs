use set_builder::set;

fn main() {
    set! { x : x <- [1, 2, 3], x + 1 };
}
