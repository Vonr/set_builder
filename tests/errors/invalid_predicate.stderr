error[E0308]: mismatched types
 --> tests/errors/invalid_predicate.rs:4:32
  |
4 |     set! { x : x <- [1, 2, 3], x + 1 };
  |                                ^^^^^ expected `bool`, found integer
